    @Test
    public void testAuthFailureShouldNotLog() throws Exception {
        doMessageTest("Jan 7 12:42:48 192.168.0.1 su[25856]: pam_authenticate: Authentication failure",
                      "192.168.0.1",
                      "uei.opennms.org/syslogd/system/Debug",
                      "pam_authenticate: Authentication failure");
    }

